### ISSUE TITLE
Integration keeps showing 'done' movies

### ISSUE THREAD

**Issue by bobvmierlo** [youdroid/home-assistant-couchpotato#2]
You're integration keeps showing the 'done' movies. Even when using the 'active' setting in the configuration.yaml file.
I've tried the following:
```
  - platform: couchpotato
    name: Wanted movies
    token: MYAPIKEY/media.list/?t=8z7yh7e3&type=movie
    host: 192.168.2.40
    state: 'active'
```
Which gives me my 'done' movies as a result.

I've also tried:
```
  - platform: couchpotato
    name: Wanted movies
    token: MYAPIKEY/media.list/?t=fsRP2gF1&type=movie&status=active
    host: 192.168.2.40
    #state: 'active'
```
Which gives a state 'unknown' in HA

**Comment by youdroid** [comment]
First of all, there is something wrong with your couchpotato integration configuration.
The token parameter must only contain the token, you don't need the end of the url.

Try :
token: MYAPIKEY ~~/media.list/?t=fsRP2gF1&type=movie&status=active~~

It may be my fault, the wiki page may not be explicit enough.
Can you try that and give me a feedback ?

**Comment by bobvmierlo** [comment]
Did that. State unknown for sensor and error below:

```
Log Details (ERROR)
Logger: homeassistant.helpers.entity
Source: custom_components/couchpotato/sensor.py:98 
First occurred: 11:01:40 PM (8 occurrences) 
Last logged: 11:05:16 PM

Update for sensor.wanted_movies fails
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 279, in async_update_ha_state
    await self.async_device_update()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 472, in async_device_update
    await self.hass.async_add_executor_job(self.update)
  File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/couchpotato/sensor.py", line 98, in update
    card_items["poster"] = movie['info']['images']['poster'][0]
IndexError: list index out of range
```

**Comment by youdroid** [comment]
This error is due to a wrong response from CouchPotato App.
For your case, Couchpotato does not return any image for this film, that is why you have this error.
I can fix it, but it also means that the picture for this movie may not be displayed.

**Comment by youdroid** [comment]
I will fix it in the next few days.
I will notify you of the fix release

**Comment by youdroid** [comment]
I fixed it in [V1.1](https://github.com/youdroid/home-assistant-couchpotato/releases/tag/V1.1)
